gemfiles/spree_3_7_spree_auth_devise.gemfile

```diff
@@ -0,0 +1,13 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem 'sassc-rails'
+gem "spree", "~> 3.7.6"
```

Comment:
This will be the same as in spree 3.7 gemfile but for clarity I would change it to `gem "spree", "~> 3.7.1"` as it is in spree_3_7.gemfile.